app/views/handlers/game_handler.py

```diff
@@ -1,25 +1,24 @@
+import datetime
 import trueskill
 
-from app.models import User, Game
+from app.models import Game
 
 
-def add_game(session, winner_id, loser_id, submitted_by_id):
-    if winner_id == loser_id:
+def add_game(session, winner, loser, submitted_by_id=None, slack_user_submitted_by=None):
```

Comment:
can a game be added if submitted_by_id and slack_user_submitted_by are both None?